docs/docs/api-specification.md

```diff
@@ -97,15 +97,15 @@ fin
 Once the initial bootstrap is finished, we start `webpack-dev-server` and an express server for serving files for the development server, and for a production build, we start building the CSS then JavaScript then HTML with webpack.
 
 During these processes there are various extension points where plugins can
-intervene. All major processes have a `onPre` and `onPost` e.g. `onPreBootstrap`
+intervene. All major processes have an `onPre` and `onPost` e.g. `onPreBootstrap`
 and `onPostBootstrap` or `onPreBuild` or `onPostBuild`. During bootstrap,
 plugins can respond at various stages to APIs like `onCreatePages`,
 `onCreateBabelConfig`, and `onSourceNodes`.
 
 At each extension point, Gatsby identifies the plugins which implement the API
 and calls them in serial following their order in the site's `gatsby-config.js`.
 
-In addition to extension APIs in node, plugins can also implement extension APIs
+In addition to extension APIs in a node, plugins can also implement extension APIs
```

Comment:
This is the only time in the document that I had a doubt about whether the writer means "node" as in "a data object" or "Node" as in "Node.js". Do you know?

Reply:
The writer means "node" as in "a data object". Thanks @shannonbux